evap/results/fixtures/minimal_test_data.json

```diff
@@ -0,0 +1,669 @@
+  "model": "evaluation.contribution", 
+  "fields": {
+    "contributor": 7, 
+    "course": 1, 
+    "responsible": false, 
+    "questionnaires": [
+      1
+    ], 
+    "can_edit": true
+  }
+},
+{
+  "pk": 4, 
+  "model": "evaluation.contribution", 
+  "fields": {
+    "contributor": 6, 
+    "course": 1, 
+    "responsible": false, 
+    "questionnaires": [
+      1
+    ], 
+    "can_edit": false
+  }
+},
+{
+  "pk": 5, 
+  "model": "evaluation.contribution", 
+  "fields": {
+    "contributor": null, 
+    "course": 2, 
+    "responsible": false, 
+    "questionnaires": [
+      2
+    ], 
+    "can_edit": false
+  }
+},
+{
+  "pk": 6, 
+  "model": "evaluation.contribution", 
+  "fields": {
+    "contributor": 2, 
+    "course": 2, 
+    "responsible": true, 
+    "questionnaires": [
+      1
+    ], 
+    "can_edit": true
+  }
+},
+{
+  "pk": 7, 
+  "model": "evaluation.contribution", 
+  "fields": {
+    "contributor": 7, 
+    "course": 2, 
+    "responsible": false, 
+    "questionnaires": [
+      1
+    ], 
+    "can_edit": true
+  }
+},
+{
+  "pk": 8, 
+  "model": "evaluation.contribution", 
+  "fields": {
+    "contributor": 6, 
+    "course": 2, 
+    "responsible": false, 
+    "questionnaires": [
+      1
+    ], 
+    "can_edit": false
+  }
+},
+{
+  "pk": 1, 
+  "model": "evaluation.course", 
+  "fields": {
+    "name_de": "a new course", 
+    "_participant_count": 25, 
+    "degree": "a degree", 
+    "voters": [], 
+    "semester": 1, 
+    "last_modified_time": "2014-09-16T23:01:13.059", 
+    "_voter_count": 17, 
+    "state": "published", 
+    "last_modified_user": 1, 
+    "participants": [
+      5, 
+      8
+    ], 
+    "vote_end_date": "2014-09-30", 
+    "name_en": "a course", 
+    "kind": "a type", 
+    "vote_start_date": "2014-09-10"
+  }
+},
+{
+  "pk": 2, 
+  "model": "evaluation.course", 
+  "fields": {
+    "name_de": "a prepared course", 
+    "_participant_count": 10, 
+    "degree": "a degree", 
+    "voters": [], 
+    "semester": 1, 
+    "last_modified_time": "2014-09-16T23:01:13.059", 
+    "_voter_count": 5, 
+    "state": "published", 
+    "last_modified_user": 1, 
+    "participants": [
+      5, 
+      8
+    ], 
+    "vote_end_date": "2014-09-30", 
+    "name_en": "a prepared course", 
+    "kind": "a type", 
+    "vote_start_date": "2014-09-10"
+  }
+},
+{
+  "pk": 1, 
+  "model": "evaluation.faqsection", 
+  "fields": {
+    "order": 0, 
+    "title_de": "testsection", 
+    "title_en": "testsection"
+  }
+},
+{
+  "pk": 1, 
+  "model": "evaluation.faqquestion", 
+  "fields": {
+    "section": 1, 
+    "order": 0, 
+    "question_de": "testquestion", 
+    "question_en": "testquestion", 
+    "answer_de": "testanswer", 
+    "answer_en": "testanswer"
+  }
+},
+{
+  "pk": 1,
+  "model": "auth.Group",
+  "fields": {
+    "name": "Staff"
+  }
+},
+{
+    "pk": 1,
+    "model": "rewards.semesteractivation",
+    "fields": {
+        "semester": 1,
+        "is_active": false
+    }
+}
+]
```

Comment:
if this is copied from the other minimal_test_data.json, please remove this one and use the other one. if not, please remove all the data that is not used for the tests.